src/main.js

```diff
@@ -42,7 +42,7 @@ export const isInstanceOfHeaders = (val) => {
  * @property {String} params - The header params as string
  */
 
-const getHeaderString = (name, val) => ` -H "${name}: ${val.replace(/(\\|")/g, '\\$1')}"`;
+const getHeaderString = (name, val) => ` -H "${name}: ${(val || '').replace(/(\\|")/g, '\\$1')}"`;
```

Comment:
I'd probably use `` `${val}`.replace `` instead of `(val || '').replace` to be consistent with the implementation of node-fetch and the browsers fetch api.

Reply:
Interesting, thanks for linking the issue, wasn't aware of browser implementation. I'll change it right away!

Reply:
Thanks, this fixes #35 now :) I'll keep #35 open until we have some regression tests.